src/lib/core/fiber.h

```diff
@@ -372,7 +372,7 @@ fiber_time(void);
  * Report loop begin time as 64-bit int.
  * Uses real time clock.
  */
-API_EXPORT uint64_t
+API_EXPORT int64_t
 fiber_time64(void);
```

Comment:
Don't know, whether it is legal to change the public C API this way. If we'll decide to do so, the actions below should be done.

I have all organization repositories cloned (at 2020-08-17), so grepped over them.

The function is used in tarantool/memcached: we should look at the usages and ensure we'll not introduce any problem by the change (building / running against old / new tarantool in any combination). Data representation is the same, sure, but if a new warning will appear, we should ensure that all builds are made without -Werror.

The function is listed in the C API webpage on the website: we should request the type change from the documentation team.

Reply:
Grepped `fiber_time64()` and `clock_time64()` (C functions) over tarantool organization (dump from April, 16). The only module that uses those functions is tarantool/memcached. Even if it'll lead to a warning, that's minor and easy to fix.